l2/system-contracts/DefaultAccount.sol

```diff
@@ -37,16 +37,29 @@ contract DefaultAccount is IAccount {
 		_;
 	}
 
+	/// @notice Validates the transaction & increments nonce. 
+	/// @dev The transaction is considered accepted by the account if 
+	/// the call to this function by the bootloader does not revert 
+	/// and the nonce has been set as used.
+	/// @param _suggestedSignedHash The suggested hash of the transaction to be signed by the user.
+	/// This is the hash that is signed by the EOA by default.
+	/// @param _transaction The transaction structure itself.
+	/// @dev Besides the params above, it also accepts unused first paramter "_txHash", which
+	/// is the unique (canonical) hash of the transaction.  
 	function validateTransaction(
-		bytes32, // _txHash 
+		bytes32, // _txHash
 		bytes32 _suggestedSignedHash, 
 		Transaction calldata _transaction
 	) external payable override ignoreNonBootloader {
 		_validateTransaction(_suggestedSignedHash, _transaction);
 	}
 
+	/// @notice Inner method for validating transaction and increasing the nonce
+	/// @param _suggestedSignedHash The hash of the transaction signed by the EOA
+	/// @param _transaction The transaction.
 	function _validateTransaction(bytes32 _suggestedSignedHash, Transaction calldata _transaction) internal {
```

Comment:
`_suggestedSignedHash` could be named slightly more explicitely? if it's something only used for EOAs, maybe have that be understandable form the name

Reply:
Well, while it is used for EOAs by default, it is generally a hash that signs over all meaningful fields of such a transaction. This hash could be used by other types of accounts if they are not 100% sure what tx hash to use

Reply:
Where do I go ? If it's being copied completely over?